### ISSUE TITLE
Lack of Cirrus and Pilatus Models in GA Aircraft

### ISSUE THREAD

**Issue by jaialin** [openscope/openscope#2007]
Unrepresented in the GA aircraft

- [SR20](https://en.wikipedia.org/wiki/Cirrus_SR20)
- [SR22](https://en.wikipedia.org/wiki/Cirrus_SR22)
- [SF50](https://en.wikipedia.org/wiki/Cirrus_Vision_SF50)
- [PC6T](https://en.wikipedia.org/wiki/Pilatus_PC-6_Porter)
- [PC12](https://en.wikipedia.org/wiki/Pilatus_PC-12)
- [PC24](https://en.wikipedia.org/wiki/Pilatus_PC-24)

**Comment by MarylandFlyer** [comment]
We do have the PC12. Would love to see the others added, though.

**Comment by jaialin** [comment]
> We do have the PC12. 

I'm not seeing PC12.json in your 

- https://github.com/MarylandFlyer/openscope/tree/develop/assets/aircraft 
- https://github.com/openscope/openscope/pull/1807/files

I'll use your n.json file though. Thanks for putting in that work.

**Comment by MarylandFlyer** [comment]
> > We do have the PC12.
> 
> I'm not seeing PC12.json in your
> 
> * https://github.com/MarylandFlyer/openscope/tree/develop/assets/aircraft
> * https://github.com/openscope/openscope/pull/1807/files
> 
> I'll use your n.json file though. Thanks for putting in that work.

True, I missed that when I was revising that file. Just be sure to include the files for the other aircraft that I’m trying to add with that PR
